app/src/main/java/org/jellyfin/androidtv/auth/model/Server.kt

```diff
@@ -16,9 +16,16 @@ data class Server(
 	val loginDisclaimer: String? = null,
 	var dateLastAccessed: Date = Date(0),
 ) {
-	private val serverVersion = version?.let(ServerVersion::fromString)
+	val serverVersion = version?.let(ServerVersion::fromString)
 	val versionSupported = serverVersion != null && serverVersion >= ServerRepository.minimumServerVersion
 
+
+	fun isVersionEqualOrGreater(otherVersion: ServerVersion?): Boolean {
```

Comment:
I think we can make this an operator function [^1]. That way the Kotlin usages look a bit nicer.

[^1]: https://kotlinlang.org/docs/operator-overloading.html#comparison-operators

Reply:
like this?

Wouldn't a comparison overload be for comparing two objects of the same type?

`operator fun compareTo(other: ServerVersion?): Int  = serverVersion?.let { other?.compareTo(it) } ?: -1`

Reply:
The compareTo function is not restricted to the same type, same for other operators actually.